assets/css/twenty-twenty-one.scss

```diff
@@ -1481,6 +1481,10 @@ a.reset_variations {
 			border: 1px solid #ddd;
 		}
 	}
+
+	table.cart img, .woocommerce table.cart img, .woocommerce-page #content table.cart img, .woocommerce-page table.cart img {
```

Comment:
I'm a little worried that this may cause problems for plugins that add images to the cart table. How would you feel about a more specific selector like this?
```suggestion
	table.cart img.woocommerce-placeholder {
```

Reply:
Yup I am ok with that. I didn't freely come up with that, it is already being done by our woocommerce-layout.css as seen here ![](https://d.pr/i/9El3Ef+)